### PR TITLE
Update for version 3.0

### PR DESCRIPTION
**Pull request by FlorianWilhelm**
Hi, 

thanks for providing this conda package. This PR updates to version 3.0.

### CODE REVIEW

**Comment by FlorianWilhelm** [comment]
After the conda-forge issue [#6527](https://github.com/conda/conda/issues/6527) is fixed, it should be possible to unpin the conda version for appveyor.

File: .circleci/config.yml [hunk 1]
```diff
@@ -69,6 +69,6 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build__CONDA_PY_27
+      - build__CONDA_PY_34
```
Comment (ocefpaf) [inline .circleci/config.yml]: conda-forge does not build for Python 3.4 due to CI resource constraints, sorry.

PS: the feedstock boilerplate should not me changed "by hand," we use conda-smithy to re-render the feedstock and revert this change.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-About pyscaffold
+About PyScaffold
 ================
 
-Home: http://pyscaffold.readthedocs.org/
+Home: http://pyscaffold.org/
 
-Package license: BSD 3-Clause
+Package license: MIT
```
Comment (ocefpaf) [inline README.md]: The README is auto generated, we need to update this in the recipe.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

One more to remove PY27 from the matrix.

**Comment by FlorianWilhelm** [comment]
@ocefpaf Thanks for the corrections, is there anything else to do?

**Comment by ocefpaf** [comment]
> @ocefpaf Thanks for the corrections, is there anything else to do?
  

Merge :grimacing:

**Pull request merged**